addons/medical_gui/functions/fnc_updateCategories.sqf

```diff
@@ -24,8 +24,17 @@ params ["_display"];
     private _enable = GVAR(actions) findIf {_category == _x select 1 && {call (_x select 2)}} > -1;
     _ctrl ctrlEnable _enable;
 
+    private _selectedColor = [
+        profilenamespace getvariable ["GUI_BCG_RGB_R",0.13],
+        profilenamespace getvariable ["GUI_BCG_RGB_G",0.54],
+        profilenamespace getvariable ["GUI_BCG_RGB_B",0.21],
+        profilenamespace getvariable ["GUI_BCG_RGB_A",0.8]
+    ];
     private _color = [[0.4, 0.4, 0.4, 1], [1, 1, 1, 1]] select _enable;
+    _color = if (GVAR(selectedCategory) isEqualTo _category) then {_selectedColor} else {_color};
```

Comment:
```suggestion
    _color = [_color, _selectedColor] select (GVAR(selectedCategory) isEqualTo _category);
```

Reply:
Knew there was a better way to do it, thanks